### ISSUE TITLE
[Forward Output Plugin] (optionally) Add hostname and source IP to record

### ISSUE THREAD

**Issue by martin21** [fluent/fluent-bit#259]
In tests with Fluentd 0.14.15 and Fluent-Bit v0.11.4 I saw that the records collected via forwarding to fluentd had no hostname field.

First I solved this by adding the hostname to the tag, but then I found I can instruct the forward input plugin of fluentd to determine hostname of Fluent Bit client host, using:

```
# Eingabe: Fluent Bit
<source>
  type forward
  bind 0.0.0.0
  port 12225
  source_hostname_key host
</source>
```

I still think it would be good if by Fluent Bit could set the hostname by itself already. This would offload the work to determine to each client. According to the documentation, forward output plugin of Fluent Bit doesn´t support this yet.

**Comment by nokute78** [comment]
Fluent-bit v0.12 development (our current GIT master) supports to append hostname to each record with `filter_record_modifier` plugin.
Could you check it? 

The PR and document (draft) is 
#235 
https://github.com/fluent/fluent-bit-docs/blob/master/filter/record_modifier.md

The section "Append fields" is the explanation to append hostname.

**Comment by edsiper** [comment]
This is already implemented for 0.12 release (not yet available), so closing the issue.

note: if you have any questions, feel free to comment.

**Comment by q2dg** [comment]
This would be a great addition to FluentBit, too!
